celery/backends/elasticsearch.py

```diff
@@ -132,7 +137,9 @@ def _get_server(self):
             '%s:%s' % (self.host, self.port),
             retry_on_timeout=self.es_retry_on_timeout,
             max_retries=self.es_max_retries,
-            timeout=self.es_timeout
+            timeout=self.es_timeout,
+            scheme=self.scheme,
+            http_auth=(self.username, self.password),
```

Comment:
The default seems to be `http_auth=None`, so I think it would be better to use this value instead of a tuple of empty strings.

Reply:
Done